metashare/api/jobs.py

```diff
@@ -432,3 +432,20 @@ def populate_github_users(repository):
 
 
 populate_github_users_job = job(populate_github_users)
+
+
+def submit_review(*, user, scratch_org, data):
+    try:
```

Comment:
@wlonk I'm not familiar with this function signature.  What does it mean/do when the first argument is an asterisk? Also, does this read as requiring these arguments (`user`, `scratch_org` and `data`) when making on POST to `submit_review`

Reply:
@duggiemitchell The `*` indicates that all arguments _after_ it must be named keyword-arguments, not positional unnamed arguments. See https://stackoverflow.com/a/14302007/854407

This `submit_review` method is a job, which is called explicitly and runs asynchronously. So the arguments here are not necessarily the required data keys in the `POST`. To answer that question, look to the docs (https://github.com/oddbird/MetaShare/pull/164/files#diff-a357898b5c3f6edf4059c8fc10d2fbd4R70-R85) or the view (https://github.com/oddbird/MetaShare/pull/164/files#diff-190eca66921d938a7d6bc9d8d3f57eb2R267-R279) and serializer (https://github.com/oddbird/MetaShare/pull/164/files#diff-c87211a17adb1e7870ce2869e2dad621R242-R245).

In brief, the `view` defines the API endpoint that the `POST` hits, but then the data is run through a `serializer` to validate it.

Reply:
@duggiemitchell So in this case, as the endpoint is currently defined, it's expecting data e.g.:

```js
   {
     notes: string;
     status: 'APPROVE' | 'REQUEST_CHANGES' | 'COMMENT';
     delete_org_on_submit: boolean;
   }
```

I'll also add that the *name* of the view determines how the URL is exposed to the front end. So here it'd be `api_urls.scratch_org_review(<org.id>)`

Reply:
Ah OK that makes sense.  Thanks, that's helpful to know when figuring how to use these backend changes :pray:.